.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
 npm-debug.log
-
+coverage
```

Comment:
I don't think there's need for that level of shrink-wrapping - the only module currently required is `clarg` which is another thing I maintain and it has no dependencies of its own either (that's the entire point of it actually - it's a simple CLI argument parser).

While it could potentially affect tests I don't think that should be enough reason to add it to the repo quite honestly. I'm really inclined to just carry on with the simpler approach of single ignore file (as for tests I think there's no harm in including them in the actual package).